heroes/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.utils.functional import cached_property
+
+
+GROUP_CHOICES= (
+    ('human', 'Human'),
+    ('robot', 'Robot'),
+    ('monster', 'Monster'),
+)
+
+
+class Group(models.Model):
+    name = models.CharField(max_length=15, choices=GROUP_CHOICES)
+
+
+class Hero(models.Model):
+    name = models.CharField(max_length=50)
+    group = models.ForeignKey(Group, on_delete=models.CASCADE, related_name='heroes')
+    health = models.IntegerField(default=100)
+    won = models.IntegerField(default=0)
+    lost = models.IntegerField(default=0)
```

Comment:
```suggestion
    lost = models.PositiveIntegerField(default=0)
```
also with won and health field